### DISCUSSION TITLE
Superset updates for CVE's

### DISCUSSION THREAD

**Discussion by arpitkh96** [apache/superset#26146] [Q&A / Help] [answered]
Our team is using superset 2.x , i saw that various findings are not resolved with latest superset 2.x release . I wanted to know what can be done to resolve this or is there any plan to maintain patches of older major versions.
CVE's 
[CVE-2023-30608](https://github.com/advisories/GHSA-rrm6-wvj7-cwh2)
[CVE-2023-30861](https://github.com/advisories/GHSA-m2qf-hxjv-5gpq)

Update - 
I have tried force updating dependencies to patch it but it didnt go well , one thing or another breaks.

**Comment by eschutho** [comment]
Hi @arpitkh96 we continue to backport security fixes to at least one major or minor version. I'm currently working on 2.1.3 and I'll see if those patches apply cleanly to that branch.

**Reply by arpitkh96** [reply]
Thanks @eschutho  , is there any ETA for this release? Also is it being tracked in any issue / or PR ?

**Reply by eschutho** [reply]
Hi @arpitkh96 You can follow the progess in slack. It was released on Friday.

**Comment by michael-s-molina** [comment, marked as answer]
@arpitkh96 [[SIP-108] Superset Release Process](https://github.com/apache/superset/issues/26015) is currently being voted and it contains many details about our release process, including what @eschutho mentioned about supported versions. Keep in mind that Superset 3.1.0 RC1 is also currently being voted and as soon as that version is officially released we'll drop support for 2.x versions. I strongly recommend migrating to 3.0.2 to continue to get security fixes.
